pkg/storage/utils/decomposedfs/decomposedfs.go

```diff
@@ -63,7 +63,7 @@ type Tree interface {
 	// CreateReference(ctx context.Context, node *node.Node, targetURI *url.URL) error
 	Move(ctx context.Context, oldNode *node.Node, newNode *node.Node) (err error)
 	Delete(ctx context.Context, node *node.Node) (err error)
-	RestoreRecycleItemFunc(ctx context.Context, key string) (*node.Node, func() error, error)
+	RestoreRecycleItemFunc(ctx context.Context, key, targetPath string) (*node.Node, func() error, error)
```

Comment:
Can you change it to `restorePath` to maintain consistency?

Reply:
@ishank011 :+1:, done